### ISSUE TITLE
Error: Cannot find module 'source-map-support

### ISSUE THREAD

**Issue by ravi1122** [nexe/nexe#745]
Hello Everyone,

I need one help here. when I am trying to build using the same directory .exe file is working properly but when I will copy that .exe to another location(different directory) it is not working I am getting this error "Error: Cannot find module 'source-map-support". I have include node_modules using -r while building but still I am getting the same issue. tried to install a source-map-module but didn't work. Can I anyone help me to resolve this issue

**Comment by ravi1122** [comment]
I appreciate your effort in guiding us regarding this issue @audstanley, I went through the rollup documentation. it is not creating executable(.exe) file right? it just bundles all the .js file. how can I run it? Even with nexe we have to support node_modules internally in our system in some directory and use it.
We have the requirement like all the files including node_modules create the binary file(.exe) and we are able to share that binary with anyone, without installing dependencies anyone can use it.

**Comment by audstanley** [comment]
No.  It's not at all creating a single binary, just a single .js file with all the dependencies "rolled up" in your application (the single js file).  From what I read, it also excludes anything that isn't being required in your application (which is awesome for optimization).  The exclusion optimizations with rollup are likely not perfect (i can imagine), and you might be able to get away with NOT supporting the node_modules directory altogether (difficult to say until you test rollup).  I've been really trying to add ARM6 support, but I've been at a catch 22 where in order to easily work on my project, I need nexe to compile on the architecture, and in order to work on nexe compiling on the architecture, I need nexe to compile on the architecture... I have some ideas to get around this, it's just more time consuming. I'm not a developer for nexe, but I'm really interested in trying to add some support for arm6 on the nexe project. 

Anyway, if you're **not** on a bizarre architecture, than i **highly** recommend using rollup first.  then you wont even need the node_modules folder OR node itself.  You'll have a complete, single binary that you can cross compile for windows, mac, and linux (all x64, maybe x86 as well - not sure). After you have the single .js file, you just pipe it into nexe. This project is absolutely amazing.  I'm going to be writing single binaries with nexe a lot.
